lib/Sema/CSDiagnostics.h

```diff
@@ -1924,6 +1924,29 @@ class UnableToInferClosureReturnType final : public FailureDiagnostic {
   bool diagnoseAsError();
 };
 
+/// Diagnose an attempt to reference a top-level name shadowed by a local
+/// member e.g.
+///
+/// ```swift
+/// extension Sequence {
+///   func test() -> Int {
+///     return max(1, 2)
+///   }
+/// }
+/// ```
+///
+/// Here `min` refers to a global function `min<T>(_: T, _: T)` in `Swift`
```

Comment:
Typo: `min` should be `max` according to code snippet above

Reply:
Ugh, thank you!

Reply:
Fixed!